gateway/src/service.rs

```diff
@@ -361,26 +361,36 @@ impl GatewayService {
         &self,
         project_name: ProjectName,
         account_name: AccountName,
+        is_admin: bool,
     ) -> Result<Project, Error> {
-        if let Some(row) = query("SELECT project_name, account_name, initial_key, project_state FROM projects WHERE project_name = ?1 AND account_name = ?2")
-            .bind(&project_name)
-            .bind(&account_name)
-            .fetch_optional(&self.db)
-            .await?
+        if let Some(row) = query(
+            r#"
+        SELECT project_name, account_name, initial_key, project_state 
+        FROM projects 
+        WHERE (project_name = ?1) 
+        AND (account_name = ?2 OR ?3 = true)
```

Comment:
```suggestion
        AND (account_name = ?2 OR ?3)
```
Might work :man_shrugging:

Reply:
It does, TIL :partying_face: